payments-app-extension-custom-onsite/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,27 @@
+[[extensions]]
+name = "{{ name }}"
+type = "payments_app_extension"
+handle = "{{ handle }}"
+target = "custom_onsite"
+
+buyer_label_to_locale = []
+default_buyer_label = "Custom Onsite Payments App Extension"
+merchant_label = "Custom Onsite Payments App Extension"
+start_payment_session_url = "https://api.paymentprovider.com/endpoint"
+supports_oversell_protection = false
+confirmation_callback_url = "https://api.paymentprovider.com/confirm"
+start_refund_session_url = "https://api.paymentprovider.com/refund"
+start_capture_session_url = "https://api.paymentprovider.com/capture"
+start_void_session_url = "https://api.paymentprovider.com/void"
+supported_countries = ["US"]
+supported_payment_methods = ["visa"]
+supports_3ds = false
+supports_installments = false
+supports_deferred_payments = false
+test_mode_available = true
+api_version = "2021-07"
+multiple_capture = false
+update_payment_session_url = "https://api.paymentprovider.com/update"
```

Comment:
I don't see this field in the partners UI. What's this?

Reply:
The field exists in the management class: https://github.com/Shopify/shopify/blob/main/components/payment_processing/payments_partners/app/models/payments_partners/payments_app_extensions/custom_onsite_payments_app_extension.rb#L49